pkgs/applications/networking/cluster/kubernix/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kubernix";
+  version = "v0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "saschagrunert";
+    repo = pname;
+    rev = version;
+    sha256 = "04dzfdzjwcwwaw9min322g30q0saxpq5kqzld4f22fmk820ki6gp";
+  };
+
+  cargoSha256 = "0n5k0ganbl6rwqskwdgvvxplwwzj5h90k9brkla4nq5w2rlqnrjl";
+  doCheck = false;
```

can you try to scope the tests? I under stand a lot of them fail. Something like:
```suggestion
  checkPhase = "cargo test --test config";
```

---

I would need additional requirements to run the tests. Am I able to specify those somewhere?

---

I was trying to scope the tests to just the unit tests. the config tests seem to all pass in a sandbox environment.

---

Sounds good